woocommerce/woocommerce-android
# Privacy Screen: update copy and links

Closes: #9109


### Description

This PR updates copy to the newest design as well as links to new documents.

### Testing instructions


Verify if copy matches designs **except "Usage Tracking"** which is expected (internal: pecCkj-AB-p2#comment-304)

Verify if tapping on "Web Options" navigates user to https://woocommerce.com/tracking-and-opt-outs/ and "Usage Tracking" to https://woocommerce.com/usage-tracking/

### Images/gif

| Actual | Design |
| --- | --- |
| <img width="566" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/5845095/ab189284-9891-418d-8cdd-6f7f4a7fd78c"> | ![android-settings(1)](https://github.com/woocommerce/woocommerce-android/assets/5845095/352ecd83-51cd-4ab0-bb18-24b45c198dfc) |
